dashboard/client/pages/index.tsx

```diff
@@ -1,8 +1,43 @@
-import { NextPage } from 'next'
+import { NextStatelessComponent } from 'next'
 import React from 'react'
 
-const HomePage: NextPage = () => {
-  return <div>Welcome to Next.js!</div>
+import Ticket from '../components/tickets'
+
+import axios from 'axios'
+import Layout from '../components/layout'
+
+import { getSpaces, getTickets } from '../gql/query'
+
+interface IProps {
+  spaces: any[]
+  tickets: any[]
+}
+
+const HomePage: NextStatelessComponent<IProps> = props => {
+  return (
+    <Layout spaces={props.spaces}>
+      {props.tickets.map(x => (
+        <Ticket key={x.title} ticket={x} />
+      ))}
+    </Layout>
+  )
 }
 
 export default HomePage
+
+HomePage.getInitialProps = async () => {
+  try {
+    const [
+      {
+        data: { spaces }
+      },
+      {
+        data: { tickets }
+      }
+    ] = await axios.all([getSpaces(), getTickets()])
+
+    return { spaces, tickets }
+  } catch (e) {
+    throw new Error(e)
```

Comment:
Let's `console.error(e.message)`, instead of throwing a new Error 🙊

Reply:
lmfao ur precommit stuff wouldnt let me console.error, that was what i had originally 🙄

Reply:
add this to `tslint.config`:

```JSON
{
    "rules": {
        "no-console": false
    }
}
```